shared/modules/conversion.utils.ts

```diff
@@ -0,0 +1,219 @@
+/**
```

Comment:
Which function is this JSDoc block for? I'm curious if we need it anymore. Or at least it seems we don't need the list of params here.

Reply:
Removed